rca/navigation/models.py

```diff
@@ -7,8 +11,50 @@
 from wagtail.core.fields import StreamField
 
 
+def url_or_relative_url_validator(value):
+    if not urlparse(value).netloc:
+        value = urljoin("https://rca.ac.uk/", value)
```

Comment:
I'd add the comment that this domain is just for validation-sake and it won't actually be used anywhere, I hope I am right?